### DISCUSSION TITLE
[자료공유] 코딩 유투버 정리

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#41] [04_자료공유]
여러분 안녕하세요!

멘토 유혁준입니다. :)

오늘은 수업 외적으로 코딩 유투버들을 소개해드리고자 합니다.

유투브에 보면 코딩 관련한 유투버들이 굉장히 많은데, 여러분들에게 도움이 될 까 싶어 정리합니다.
지금부터 적는 유투버님들은 제가 정말 존경하고 뉴스보듯이 자주보는 채널입니다ㅎㅎ

- [조코딩](https://www.youtube.com/channel/UCQNE2JmbasNYbjGAcuBiRRg) : 비전공자 개발자이시며, 현재는 코드라이언 코딩 강의 및 `web, python`에 대한 강의를 하고 계십니다. 
- [드림코딩 by 엘리](https://www.youtube.com/channel/UC_4u-bXaba7yrRz_6x6kb_w) : 정말 뛰어난 프론트엔드 개발자이시며, 현재 마이크로소프트 엣지의 프론트엔드를 담당하고 계십니다. `web programming` 공부하시는데 정말 너무나도 좋은 강의자료 및 수업을 올려주시고 계십니다. 저도 이분 `자바스크립트` 강의를 들었는데, 정말 좋습니다. 궁금하시면 [여기](https://github.com/yoomambo/Javascript_Basic) 들어가보시길 바랄게요.
- [노마드코더](https://www.youtube.com/channel/UCUpJs89fSBXNolQGOYKn0YQ) : 콜롬비아 사람인 니콜라스 선생님의 채널입니다. 전반적인 프로그래밍 관련 기본 basic 한 지식부터 시작해서, 전문적인 `web programming` 강의까지. 여러분들의 전반적인 프로그래밍 지식을 정말 쉽게 말씀해주십니다. 무조건 추천!!!
- [코딩하는 테크보이 워니](https://www.youtube.com/channel/UC0uDM1xZMNBAoW2xnzhAQ7g) : 현직 개발자이시며, `개발자에 관련된 정보 뿐만아니라 프로그래밍 지식` 도 습득 가능한 채널입니다.
- [나도코딩](https://www.youtube.com/channel/UC7iAOLiALt2rtMVAWWl4pnw) : 기본적으로 프로그래밍 언어 관련된 강의를 주로 올리십니다. `C, 파이썬` 관련된 강의를 주로 올리십니다.
- [생활코딩](https://www.youtube.com/user/egoing2) : 주로 `web programming` 관련 수업을 올리십니다. 요즘은 `머신러닝, github, SQL, JAVA, Javascript, NodeJS` 등 정말 많은 강의도 업로드 하시니까 한 번 들어가보세요~
- [Sung Kim](https://www.youtube.com/channel/UCML9R2ol-l0Ab9OXoNnr7Lw) : 전 홍콩과기대 교수님, 현재는 네이버 AI 책임자를 맡고 계십니다. `머신러닝 및 딥러닝, 강화학습` 관련된 강의를 주로 올리십니다.
- [개발자 라라](https://www.youtube.com/channel/UCjWnBKjlXv550KDiBjtW8vQ) : `코딩 관련된 기본 지식, 개발자 관련 이슈, 여러 현업에서 발생하는 이슈` 들을 정리하십니다. 기본적인 background를 보다 쉽게 얻을 수 있어요
- [삼평동연구소](https://www.youtube.com/channel/UC7fEWgloU6nxYS-uI-fvloQ) : 판교 테크노벨리에서 일하시는 `현업자들의 이야기` 를 들을 수 있는 채널입니다.
- [코딩factory](https://www.youtube.com/channel/UCWl-upL6tX9ct4Airq4KeHg) : 프로그래밍으로 여러가지 `예시` 를 만드셔서 적용하는 유투버입니다.
- [홍정모의 실리콘밸리 도전기](https://www.youtube.com/channel/UCg6IlhycdYiK_nWB3spjIqA) : 동국대 컴퓨터공학과 교수님이었다가 현재는 facebook 개발자로 들어가신 홍정모 교수님입니다. 교수님답게 `C, C++, 컴퓨터구조, 기술동향 분석` 등 여러가지 프로그래밍 관련 지식을 강의로 올려주시는 아주 좋은 채널입니다.

제가 지금까지 구독하는 채널입니다. 알림을 설정해놔서 뉴스처럼 보는데 정말 좋은 채널이라고 생각해요~

여러분들도 혹시 위의 채널말고 다른 좋은 채널있으면 공유부탁해요~~

감사합니다.

멘토 유혁준

**Comment by lsj0822** [comment]
https://www.youtube.com/c/cmiscm/videos (인터렉티브 디벨로퍼)

저는 이분 추천해드려요!  미대생이지만 웹디자인 포트폴리오로 구글 입사하신 분인데, HTML5에 관한 프로그래밍 지식뿐만 아니라 아이디어와 그걸 실현시키는 방법이 좋아서 자주보는 채널이에요

**Reply by yoomambo** [reply]
오... 반응형 웹페이지 관련된 콘텐츠를 주로 다루시네요!
제가 몰랐던 채널이라 바로 구독했어요 ㅎㅎ 이런 정보공유 아주 좋아요!
